src/main/resources/templates/components/form.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="ja" xmlns:th="http://www.thymeleaf.org">
+
+  <div th:fragment="field-text(fieldid, labelname, required)" class="field">
+    <div th:with="hasErrors=${#fields.hasErrors('__${fieldid}__')}">
+      <label class="label" th:for="${fieldid}">
+        <span th:if="${required}" th:text="|*|" class="has-text-danger"></span>
+        [[${labelname}]]
+      </label>
+      <div class="control">
+        <input class="input" type="text" th:id="${fieldid}" th:name="${fieldid}" th:value="*{__${fieldid}__}"
+            th:classappend="${hasErrors} ? is-danger">
+      </div>
+      <p class="help is-danger" th:if="${hasErrors}" th:errors=*{__${fieldid}__}></p>
+    </div>
+  </div>
+
+  <div th:fragment="field-password(fieldid, labelname, required)" class="field">

```

Comment:
This might be unified into `input-text`.

Reply:
I thought it needed to clear password field before returning view.
That will make implementation more complicated, so I didn't do that.